Build/Configuration.cs

```diff
@@ -4,11 +4,9 @@
 [TypeConverter(typeof(TypeConverter<Configuration>))]
 public class Configuration : Enumeration
 {
-    public static Configuration Debug { get; } = new() { Value = nameof(Debug) };
+    public static Configuration Debug = new() { Value = nameof(Debug) };
 
-    public static Configuration Release { get; } = new() { Value = nameof(Release) };
-
-    public static Configuration CI { get; } = new() { Value = nameof(CI) };
+    public static Configuration CI = new() { Value = nameof(CI) };
```

Comment:
This change triggers several analyzers for me :suspect: 
[SA1401](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1401.md)
[MA0069](https://github.com/meziantou/Meziantou.Analyzer/blob/main/docs/Rules/MA0069.md)

Reply:
Weird. I'm not getting those. I can ignore those, but I'm wondering why you are getting them and I don't.

Nuke needs those to be fields, otherwise you can't pass them as CLI configuration parameters or environment variables.

Reply:
> Weird. I'm not getting those. I can ignore those, but I'm wondering why you are getting them and I don't.

I didn't get them on a regular build of `_build` but only when manually triggering code analysis.

> Nuke needs those to be fields, otherwise you can't pass them as CLI configuration parameters or environment variables.

Aha... 
It's [`[TypeConverter(typeof(TypeConverter<Configuration>))]`](https://github.com/nuke-build/nuke/blob/6b080532916f16742454c41e8534cc64e3608c63/source/Nuke.Tooling/Enumeration.cs#L101) that looks for fields but not properties.

Let's suppress the analyzers: SA1401, S1104, S2223, MA0069 and move on.

Reply:
Discussed 1:1 - Since the analyzers aren't triggered automatically I'll take care of this later separately from this PR.